guides/common/modules/proc_adding-an-http-proxy.adoc

```diff
@@ -12,6 +12,7 @@ To use the CLI instead of the {ProjectWebUI}, see the xref:cli-adding-an-http-pr
 . In the {ProjectWebUI}, navigate to *Infrastructure* > *HTTP Proxies* and select *New HTTP Proxy*.
 . In the *Name* field, enter a name for the HTTP proxy.
 . In the *URL* field, enter the URL for the HTTP proxy, including the port number.
+ifdef::satellite[]
```

Comment:
I don't think this is right. The connections to the CDN are needed to retrieve RHEL content. That's why it's marked as generic.

It may be Katello-specific since non-Katello RHEL users will probably use it straight from the internet, but I'm not quite sure. Those are rare (turns out provisioning RHEL on non-Katello was broken since Foreman 2.5).

Reply:
Dropped this commit.